gpkitmodels/GP/aircraft/wing/wing.py

```diff
@@ -81,15 +72,19 @@ def setup(self, static, state):
         Re = Variable("Re", "-", "Reynold's number")
         cdp = Variable("c_{dp}", "-", "wing profile drag coeff")
 
-        path = os.path.dirname(__file__)
-        df = pd.read_csv(path + os.sep + "jho_fitdata.csv")
+        df = pd.read_csv(fitdata)
         fd = df.to_dict(orient="records")[0]
 
+        if fd["d"] == 2:
+            independentvars = [CL, Re]
+        elif fd["d"] == 3:
+            independentvars = [CL, Re, static["\\tau"]]
+
         constraints = [
             Cd >= cdp + CL**2/np.pi/static["AR"]/e,
             Re == state["\\rho"]*state["V"]*static["c_{MAC}"]/state["\\mu"],
             # XfoilFit(fd, cdp, [CL, Re], airfoil="jho1.dat"),
-            XfoilFit(fd, cdp, [CL, Re]),
+            XfoilFit(fd, cdp, independentvars),
```

Comment:
I'd rather these be passed in as optional arguments I think....`Cl=, Re=, tau=` etc